packages/angular-output-target/__tests__/plugin.spec.ts

```diff
@@ -13,19 +13,19 @@ describe('normalizeOutputTarget', () => {
     }).toThrow(new Error('rootDir is not set and it should be set by stencil itself'));
   });
 
-  it('should return fail if proxiesFile is not set', () => {
+  it('should throw an error if proxyDeclarationFile is not set', () => {
```

Comment:
Since `directivesProxyFile` is deprecated but still supported, should we have separate tests for `directivesProxyFile` and `proxyDeclarationFile` instead?

Reply:
I think updating the test description here may make the most sense:
> should throw an error if proxyDeclarationFile and directivesProxyFile is not set

Since the check for throwing an error is dependent on both `directivesProxyFile` and `proxyDeclarationFile` being null/undefined, there is no error behavior to test if only one is defined/undefined.